### DISCUSSION TITLE
Blender hanging when trying to run nxt editor

### DISCUSSION THREAD

**Discussion by shhlife** [nxt-dev/nxt_editor#154] [Q&A]
Heya,

I'm trying to use nxt with Blender 2.91.2 on OS X. I'm able to install it just fine, and if I run nxt from the terminal it comes up - but Blender hangs when trying to run nxt after using the addon.

I've let it sit for over a half hour and it doesn't appear to be doing anything. No error messages in the shell I launched Blender from, either.

<img width="800" alt="image" src="https://user-images.githubusercontent.com/5397345/106371996-1618bf00-63d0-11eb-89e4-b6c249252400.png">

Any tips?
thanks!
-Jason

**Comment by ImLucasBrown** [comment]
Hey Jason, sorry for the delay, my MacOS machine is off getting a new logic board 😩
@MichaelAldrich might be able to boot it up in MacOS at it the meantime.

Can you try running this bit of Python in Blender's interactive interpreter and see if any errors print out?
```python
import os
os.environ['NXT_VERBOSE'] = '2'
os.environ['NXT_DCC'] = 'blender'

import nxt_editor

nxt_editor._new_qapp()
nxt_editor.show_new_editor()
```

**Comment by MichaelAldrich** [comment]
Thanks for the report @shhlife , I just reproduced it on my own MacOS machine.

I opened an issue for this here: https://github.com/nxt-dev/nxt_editor/issues/156 and used your screenshot 😉 
We'll do our best to get a fix in the next release.

**Reply by MichaelAldrich** [reply]
Good news! In the latest release of the editor (3.7.0) this appears to be fixed on my Mac. @shhlife if you get a chance to try, please tell us how it goes for you.
